src/components/zenlytic-legend/LegendIcon.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+export const LegendIcon = ({ color, type = 'line' }) => {
+  if (type === 'line') {
+    return (
+      <svg
+        className="recharts-surface"
+        width="12"
+        height="12"
+        viewBox="0 0 32 32"
+        version="1.1"
+        style={{ display: 'inline-block', verticalAlign: 'middle', marginRight: '4px' }}>
+        <path
+          strokeWidth="4"
+          fill="none"
+          stroke={color}
+          d="M0,16h10.666666666666666
```

Comment:
This is ripped from Recharts, it's the icon we use below: 
<img width="78" alt="image" src="https://user-images.githubusercontent.com/72046909/217033561-2cb3261d-56bd-4c2a-b73a-57cadcbe06a5.png">

It doesn't appear to be exported from the Recharts library, so I'm not sure of a better way if we want to use the same icon in this legend.